test/Flick/AnimatedCarousalPage.xaml.cs

```diff
@@ -49,6 +57,76 @@ public sealed partial class AnimatedCarousalPage : Page
         public AnimatedCarousalPage()
         {
             this.InitializeComponent();
+
+            ElementCompositionPreview.GetElementVisual(sv).Clip = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(sv).Compositor.CreateInsetClip();
+        }
+
+        private object selectedItem = null;
+
+        public object SelectedItem
+        {
+            get
+            {
+                return selectedItem;
+            }
+            set
+            {
+                selectedItem = value;
+            }
+        }
+
+        private bool isScrolling = false;
+
+        public bool IsScrolling
+        {
+            get
+            {
+                return isScrolling;
+            }
+        }
+
```

Comment:
private bool IsScrolling {get;set;} should work. Although I'm not quite sure if this is needed.

Reply:
Not sure either. I added it when I was debugging the animated scrolling issues we experienced when calling ScrollToCenterOfViewport. I can do some testing in a later iteration to assess necessity.